pkg/service/test/util.go

```diff
@@ -271,6 +298,11 @@ func filterTcsMocks(tc *models.TestCase, m []*models.Mock, logger *zap.Logger) [
 	}
 
 	for _, mock := range m {
+		if mock.Version == "api.keploy-enterprise.io/v1beta1" {
+			logger.Info("This mock was recorded using the the enterprise version, may not work properly with the open source version", zap.String("mock kind:", string(mock.Kind)))
```

Comment:
Please remove the typo here, and can the info be logged once for a test-set.

Reply:
Done